lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/Inputs.pm

```diff
@@ -9,74 +9,60 @@ use Genome::InstrumentData::Command::Import::WorkFlow::SourceFiles;
 
 class Genome::InstrumentData::Command::Import::WorkFlow::Inputs { 
     is => 'UR::Object',
-    has_transient => {
-        incoming_params => { is => 'HASH', },
-        instrument_data_properties => { is => 'HASH', },
+    has => {
+        analysis_project => { is => 'Genome::Config::AnalysisProject', },
+        library => { is => 'Genome::Library', },
+        instrument_data_properties => { is => 'HASH', default_value => {}, },
         source_files => { is => 'Genome::InstrumentData::Command::Import::WorkFlow::SourceFiles', },
+    },
+    has_transient => {
```

Comment:
I don't think its necessary for these `via`/`to` properties to be `transient` if the properties they are `via` are not.